app/views/layouts/_search_form.html.slim

```diff
@@ -4,5 +4,5 @@ div class="relative" data-controller="search" data-search-version="#{ruby_versio
     div class="relative"
       button type="submit" class="absolute left inset-y-0 px-4 #{'text-gray-700 fill-current' if homepage?}" aria-label="search"
         span data-target="search.button" class="fa fa-search"
-      input id="search" name="q" data-target="search.input" data-action="keyup->search#autocomplete" type="text" autocapitalize="off" autocorrect="off" autocomplete="off" placeholder="Search Ruby Documentation" aria-label="Search" title="Search" value=(search_query unless homepage?) class="#{homepage? ? 'bg-white shadow-md py-3 z-0 text-gray-800 placeholder-gray-600' : 'bg-red-800 focus:bg-white focus:text-black dark-focus:bg-white dark-focus:text-black dark:bg-gray-800 py-2 placeholder-white'} rounded px-3 pl-10 py-2 w-full outline-none placeholder-opacity-100" *{homepage? ? 'autofocus' : '' => ''}
+      input id="search" name="q" data-target="search.input" data-action="keydown->search#on_keydown" type="text" autocapitalize="off" autocorrect="off" autocomplete="off" placeholder="Search Ruby Documentation" aria-label="Search" title="Search" value=(search_query unless homepage?) class="#{homepage? ? 'bg-white shadow-md py-3 z-0 text-gray-800 placeholder-gray-600' : 'bg-red-800 focus:bg-white focus:text-black dark-focus:bg-white dark-focus:text-black dark:bg-gray-800 py-2 placeholder-white'} rounded px-3 pl-10 py-2 w-full outline-none placeholder-opacity-100" *{homepage? ? 'autofocus' : '' => ''}
```

Comment:
https://stimulus.hotwire.dev/reference/actions#naming-conventions

Per the StimulusJS handbook, action names should be camel case, since they're JS functions.

```suggestion
      input id="search" name="q" data-target="search.input" data-action="keydown->search#onKeydown" type="text" autocapitalize="off" autocorrect="off" autocomplete="off" placeholder="Search Ruby Documentation" aria-label="Search" title="Search" value=(search_query unless homepage?) class="#{homepage? ? 'bg-white shadow-md py-3 z-0 text-gray-800 placeholder-gray-600' : 'bg-red-800 focus:bg-white focus:text-black dark-focus:bg-white dark-focus:text-black dark:bg-gray-800 py-2 placeholder-white'} rounded px-3 pl-10 py-2 w-full outline-none placeholder-opacity-100" *{homepage? ? 'autofocus' : '' => ''}
```

Reply:
Fixed in b994731d3f9d6123ce47bb73f33da1b9136ff55d

Reply:
Thanks!